sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/kubernetescluster/samples/DeployImageFromContainerRegistryToKubernetes.java

```diff
@@ -144,8 +145,9 @@ public static boolean runSample(AzureResourceManager azureResourceManager, Strin
                 .withServicePrincipalClientId(servicePrincipalClientId)
                 .withServicePrincipalSecret(servicePrincipalSecret)
                 .defineAgentPool("agentpool")
-                    .withVirtualMachineSize(ContainerServiceVMSizeTypes.STANDARD_D1_V2)
+                    .withVirtualMachineSize(ContainerServiceVMSizeTypes.STANDARD_D2_V2)
```

Comment:
Why vm size changes here?

Reply:
The `SYSTEM` agent pool doesn't support 1 cpu VM